task/build-image-index/0.1/build-image-index.yaml

```diff
@@ -147,3 +162,140 @@ spec:
       capabilities:
         add:
           - SETFCAP
+
+  - image: quay.io/redhat-appstudio/sbom-utility-scripts-image:d4935b35eb0849841412d5d1e481a812e939f2bf
+    name: create-sbom
+    computeResources:
+      limits:
+        memory: 512Mi
+        cpu: 200m
+      requests:
+        memory: 256Mi
+        cpu: 100m
+    env:
+    - name: IMAGE_URL
+      value: $(results.IMAGE_URL.path)
+    - name: IMAGE_DIGEST
+      value: $(results.IMAGE_DIGEST.path)
+    script: |
+      TOADD="${IMAGE_URL}@${IMAGE_DIGEST}"
+      buildah manifest inspect "$TOADD" > manifest_data.json
+      python3 /scripts/index-image-sbom-script.py \
+        --image-index-url "$IMAGE_URL" \
+        --image-index-digest "$IMAGE_DIGEST" \
+        --inspect-input-file manifest_data.json
+        --output-path sbom-results.json
+    workingDir: $(workspaces.source.path)
+    securityContext:
+      runAsUser: 0
+
+  - name: inject-sbom-and-push
```

Comment:
What is this step used for in the context of the index image? Do we need it here?

Reply:
I thought about this step a little bit more and I think it is not relevant for the index image. This step seems to add sbom into the actual image a push it to the registry. The index image is not an actual image (it doesn't have any layer). It is just a manifest that contains references to a child's images. I think you can remove it completely and also get rid of additional params that you added to the task.